cvat/apps/dataset_manager/bindings.py

```diff
@@ -546,14 +623,18 @@ def convert_attrs(label, cvat_attrs):
                 a_value = cvat_attrs.get(a_name, a_desc['default_value'])
                 try:
                     if a_desc['input_type'] == AttributeType.NUMBER:
-                        a_value = float(a_value)
+                        if self._dimension == DimensionType.DIM_2D:
```

Comment:
How it depends on the task dimension?

Reply:
This check is not required now.
Reverted changes